FiskalySDK/Classes/JsonRpc/JsonRpcError.swift

```diff
@@ -13,16 +13,27 @@ public class JsonRpcError: Error, Codable {
     }
 
     public required init(from decoder: Decoder) throws {
```

Comment:
Hm, would it make sense to turn the entire type into a struct and get the synthesized initializer for free?

Reply:
Good point, I don't think there is much need for this, unless the coding keys are likely to change. It would make sense for JsonRpcResponse too.